git-config/main.tf

```diff
@@ -44,18 +44,31 @@ data "coder_parameter" "username" {
   name         = "username"
   type         = "string"
   default      = ""
-  description  = "Git user.name to be used for commits. Leave empty to default to Coder username."
+  description  = "Git user.name to be used for commits. Leave empty to default to Coder user's Full Name."
   display_name = "Git config user.name"
```

Comment:
Do you think this display name is most intuitive?

Or should it be replaced with something like "Full name for Git config"?

Reply:
Hmm yeah. We can replace it.